css/main.css

```diff
@@ -588,8 +588,8 @@ main.content {
 
 @media only screen and (min-width: 770px) {
   #book-selection-menu-chapter-list {
-    width: 50.5em;
-    min-height: 20.8em;
+    width: 50.6em;
```

Comment:
@tobias-klein have you tried to achieve the same width for book and chapter selection?

Reply:
Yes, that's what I tried. I'll have another look to see how this works with different languages. In general, it's difficult to replicate the width of the book selection to the chapter selection, because both elements use completely different layout methods ...

Reply:
I tested this now with multiple languages and currently it is not as it should be. The widths vary based on the languages used. And you cannot really calculate the overall width of the menu as before (I think before the column widths were static).

Another way would be maybe to not hide the book menu completely (`display: none`) when showing the chapter selection, but rather using `visibility: hidden`  and overlaying the chapter menu somehow using relative positioning.

Any additional suggestions?

Reply:
How about if I move the width setting to the `#book-selection-menu` container?